### PR TITLE
[ISSUE #2402] Update directory for RocketMQ_Example.md

### PR DESCRIPTION
**Pull request by nianjiang**
## What is the purpose of the change

添加一个目录，便于对文章的架构快速了解。

## Brief changelog

使用这个工具，自动生成目录：
https://github.com/ekalinin/github-markdown-toc

### CODE REVIEW

**Comment by RongtongJin** [comment]
Good Job！Plz create a issue and link this pr.

**Comment by nianjiang** [comment]
create issue：https://github.com/apache/rocketmq/issues/2402

File: docs/cn/RocketMQ_Example.md [hunk 1]
```diff
@@ -1,4 +1,43 @@
 # 样例
+-----
+ * [目录](#样例)
+      * [1 基本样例](#1-基本样例)
+         * [1.1 加入依赖：](#11-加入依赖)
```
Comment (RongtongJin) [inline docs/cn/RocketMQ_Example.md:5]: There is an extra colon here. [1.1 加入依赖：] -> [1.1 加入依赖]

**Comment by vongosling** [comment]
Very impressive improvement !

**Pull request merged**